translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/ProjectManager/activities/ActivityChapterList.java

```diff
@@ -110,7 +108,7 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
 
         // Set its layout manager
         mLayoutManager = new LinearLayoutManager(this);
-        mLayoutManager.setOrientation(LinearLayoutManager.VERTICAL);
+        mLayoutManager.setOrientation(RecyclerView.VERTICAL);
```

Comment:
https://developer.android.com/reference/android/support/v7/widget/LinearLayoutManager.html#setOrientation(int)

This change seems incorrect?

Reply:
@sarabiaj My IDE underlined it as error and recommended to use RecyclerView constants instead. But I can revert it back to LinearLayoutManager.

Reply:
@sarabiaj it's back LinearLayoutManager